syncano/models/base.py

```diff
@@ -413,12 +414,22 @@ class CodeBox(Model):
     :ivar name: :class:`~syncano.models.fields.StringField`
     :ivar created_at: :class:`~syncano.models.fields.DateTimeField`
     :ivar updated_at: :class:`~syncano.models.fields.DateTimeField`
+
+    .. note::
+        **CodeBox** has special method called ``run`` which will execute attached source code::
+
+            >>> CodeBox.please.run('instance-name', 1234, payload={'variable_one': 1, 'variable_two': 2})
+
+        or via instance::
+
+            >>> cb = CodeBox.please.get('instance-name', 1234)
+            >>> cb.run(variable_one=1, variable_two=2)
     """
 
     LINKS = (
         {'type': 'detail', 'name': 'self'},
         {'type': 'list', 'name': 'runtimes'},
-        {'type': 'detail', 'name': 'run'},
+        # {'type': 'detail', 'name': 'run'},
```

Comment:
`links` field adds relations to model based on those links eg. `instance.codeboxes.all()`so in this case it we have name collision.